File: 프로그래머스/lv0/120842. 2차원으로 만들기/2차원으로 만들기.py
def solution(num_list, n):
    answer = []
    
    #n개씩 잘라서 배열에 넣기 -> for문
    #n, n+1 씩 append -> *i 해서 반복문 돌아가는 만큼 들어가도록
    #반복문 돌아가는 횟수 = numlist를 n으로 나눈 몫
    for i in range(len(num_list)//n):
        answer.append(num_list[i*n : n*(i+1)])
    return answer

File: 프로그래머스/lv0/120842. 2차원으로 만들기/2차원으로 만들기.java
class Solution {
    public int[][] solution(int[] num_list, int n) {
        int[][] answer = new int [num_list.length/n][n];
        int count = 0;
        
        for (int i = 0 ; i< num_list.length/n; i++) {
            //j: n개의 값을 answer에 넣기
            for (int j = 0; j<n; j++) {
                answer[i][j] = num_list[count];
                count++;

            }
        }
        return answer;
    }
}